### PR TITLE
fix: modal style

### LINKED ISSUE #5829: QR code modal is not fully displayed on profile page
### What happened?

QR code modal is not fully displayed in user page.

### Please reproduce in steps

1. go to https://canary.kodadot.xyz/bsx/u/bXjrHpyJzSczJuuqmtN6nqCnd5b7KFa5LK1rBtF1tUHDYzp8N?page=1
2. click qrcode button

### Expected Behavior

The screenshot below shows what the modal looks like on kodadot.xyz.  

image

### What browsers are you seeing the problem on?

Desktop Chrome

### At which address did you encounter bug?

canary.kodadot.xyz, beta.kodadot.xyz

### Are you logged in?

Yes

### Which wallet you are using?

PolkadotJS

### At which chain did you encounter bug?

_No response_

### Screenshots

image

### Relevant log output

_No response_

### Payment link for reward

https://beta.kodadot.xyz/transfer?target=GqheyaQ3c8SFBJYorTXyLW6jvLXKNAXzbad77NXoLYJBXXs

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Gavin-Gong**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5829 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer?target=GqheyaQ3c8SFBJYorTXyLW6jvLXKNAXzbad77NXoLYJBXXs)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at 41870e2</samp>

Improved the appearance and functionality of the sharing modal. Fixed a CSS selector bug that affected the QR code button. Removed a redundant z-index property from the body element.

### <samp>🤖 Generated by Copilot at 41870e2</samp>

> _We're working on the code, me hearties, yo ho ho_
> _We're fixing up the modal and the QR code, yo ho ho_
> _We're changing all the selectors and the `z-index` too_
> _We're making it look pretty and we're almost through, yo ho ho_

### CODE REVIEW

**Comment by exezbcz** [comment]
good! works now, could you please also add fill to the container so its not transparent

thanks!

**Comment by Gavin-Gong** [comment]
> good! works now, could you please also add fill to the container so its not transparent

done, I used theme background color to fill the content container

**Review by preschian — APPROVED** [review]

File: styles/global.scss [hunk 1]
```diff
@@ -10,9 +10,6 @@ body,
 div#__nuxt,
 div#__layout,
 main {
-  //without this, footer image is above main contents
-  // block hover, click events
-  z-index: 100;
```
Comment (roiLeo) [inline styles/global.scss:15]: no side effects if we remove this line?

File: components/shared/modals/ModalWrapper.vue [hunk 2]
```diff
@@ -51,8 +51,36 @@ export default class ModalWrapper extends Vue {
...
+:deep .modal-close {
+  height: 40px;
+  position: fixed;
+  right: 20px;
+  top: 20px;
+  width: 40px;
+}
+.modal-card {
+  text-align: center;
+}
```
Comment (roiLeo) [inline components/shared/modals/ModalWrapper.vue:82]: can be replaced by `has-text-centered`

**Review by roiLeo — CHANGES_REQUESTED** [review]
We need to rewrite `ModalWrapper` with new `NeoModal` component

Preview doesn't work for me
Screenshot 2023-05-02 at 08-56-52 Aksum KodaDot - NFT Market Explorer

**Review by preschian — CHANGES_REQUESTED** [review]
already pointed by @roiLeo

**Comment by preschian** [comment]
On the collection page, some components are missing when the modal is shown. is it related?

Screenshot 2023-05-03 084841

**Comment by Gavin-Gong** [comment]
> On the collection page, some components are missing when the modal is shown. is it related?
> 
> Screenshot 2023-05-03 084841

Not related，The NeoModal component incorrectly caused the Navbar component to calculate the `top` value .
image

As a workaround, setting o-modal `scroll="clip"` can solve this problem.

**Review by roiLeo — APPROVED** [review]
Nice work! 🚀 

wfm ✅ 

you might want to check #5852 as we still have some `<b-modal />` code to replace

File: components/gallery/GalleryItemTabsPanel/GalleryItemOffers.vue [hunk 3]
```diff
@@ -185,11 +185,11 @@ const formatOfferStatus = (status: OfferStatusType, expiration: number) => {
...
 
 const onWithdrawOffer = async (caller: string) => {
-  await submit(caller, ShoppingActions.WITHDRAW_OFFER, refetch)
+  await submit(caller, ShoppingActions.WITHDRAW_OFFER)
 }
 
 const onAcceptOffer = async (caller: string) => {
-  await submit(caller, ShoppingActions.ACCEPT_OFFER, refetch)
+  await submit(caller, ShoppingActions.ACCEPT_OFFER)
 }
```
Comment (vikiival) [inline components/gallery/GalleryItemTabsPanel/GalleryItemOffers.vue:193]: Why remove?

**Review by vikiival — COMMENTED** [review]
👁️

File: components/gallery/GalleryItemTabsPanel/GalleryItemOffers.vue (hunk 3 above)
Comment (Gavin-Gong) [reply components/gallery/GalleryItemTabsPanel/GalleryItemOffers.vue:193]: > Why remove?

Just to fix the warning from deep scan. The `submit` function does not actually consume the `refetch` function.

Comment (vikiival) [reply components/gallery/GalleryItemTabsPanel/GalleryItemOffers.vue:193]: Yup thanks, I spotted bug #5880

**Review by vikiival — APPROVED** [review]

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 25.62 USD/KSM ~ 1.171 $KSM
🧗 [GqheyaQ3c8SFBJYorTXyLW6jvLXKNAXzbad77NXoLYJBXXs](https://kusama.subscan.io/account/GqheyaQ3c8SFBJYorTXyLW6jvLXKNAXzbad77NXoLYJBXXs)
🔗 [0xa722078da29167811b2cfe056d4cbf6b4da8acc7e58932c5640d289316456f7d](https://kusama.subscan.io/extrinsic/0xa722078da29167811b2cfe056d4cbf6b4da8acc7e58932c5640d289316456f7d)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

File: styles/global.scss (hunk 1 above)
Comment (daiagi) [reply styles/global.scss:15]: @Gavin-Gong  I guess i'm late to the conversation and this already have been merged but

you can override the z-index of the modal:

https://oruga.io/components/Modal.html#style

check out NeoCheckbox for example how oruga scss vars can be overridden

**Pull request merged**